.github/workflows/end-to-end-test-tracer-replay_testbench.yaml

```diff
@@ -0,0 +1,41 @@
+name: end-to-end-test
```

Comment:
we don't need 2 e2e tests for tracer. Modify the original tracer replay test to also incorporate the multi threaded code tracing